ray_on_golem/server/services/ray.py

```diff
@@ -248,16 +251,25 @@ async def _monitor_node(self, node_id: NodeId, activity: Activity) -> None:
                         f"Something is wrong with node {node_id} activity. {activity_state=}"
                     )
             except Exception:
+                provider_desc = await self._golem_service.get_provider_desc(activity)
                 msg = (
-                    f"Activity on {await self._golem_service.get_provider_desc(activity)} "
+                    f"Activity on {provider_desc} "
                     f"is no longer accessible. Terminating {node_id} {is_head_node=}"
                 )
                 logger.warning(msg)
                 logger.debug(msg, exc_info=True)
                 if is_head_node:
+                    self.add_service_warning(
+                        f"{datetime.now(timezone.utc).isoformat()} Terminating cluster as "
```

Comment:
Looks like the timestamp could be moved into `add_service_warning` as a bool with a default.

Reply:
Done